docs/reserved-keywords.adoc

```diff
@@ -0,0 +1,278 @@
+= PartiQL Reserved Keywords
+
+Reserved keywords are part of PartiQL Language that are used by PartiQL to parse and understand PartiQL statements.
+PartiQL Language Grammar Reserved Keywords cannot be used in PartiQL statements as-is.
+For example, the following query results in a _Syntax Error_ because `FROM` is a Reserved Keyword in PartiQL:
+
+.Example 1. Invalid PartiQL Query with a Reserved Keyword
+[source, sql]
+----
+SELECT t.FROM FROM <<{'ID': 1, 'FROM': 'NA'}, {'ID': 2, 'FROM': 'SA'}>> AS t;
+----
+
+In order to use Reserved Keywords in your PartiQL statements, use Quoted Identifiers as Example 2 shows:
+
+.Example 2. Using Quoted Identifier as a Reserve Keyword in a PartiQL Statement.
+[source, sql]
+----
+SELECT t."FROM" FROM <<{'ID': 1, 'FROM': 'NA'}, {'ID': 2, 'FROM': 'SA'}>> AS t;
+----
+
+== Reserved Keywords
+[NOTE]
+====
+Reserved Keywords are yet to be https://github.com/partiql/partiql-spec/issues/2[formally defined] in PartiQL Specification, therefore there may be some differences in different PartiQL implementations.
+====
+
+[cols="1,1,1,1",grid=none, frame=none]
+|===
+
+|ABSOLUTE
```

Comment:
Out of curiosity, where was this initial list taken from?

Reply:
It's from here: https://partiql.org/syntax/antlr.html